embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/utils/exceptions/Unchecked.kt

```diff
@@ -1,64 +1,13 @@
 package io.embrace.android.embracesdk.utils.exceptions
 
-import io.embrace.android.embracesdk.utils.exceptions.function.CheckedSupplier
 import java.lang.reflect.InvocationTargetException
 
-/**
- * Static utility methods that convert checked exceptions to unchecked.
- *
- * Two `wrap()` methods are provided that can wrap an arbitrary piece of logic
- * and convert checked exceptions to unchecked.
- *
- * A number of other methods are provided that allow a lambda block to be decorated
- * to avoid handling checked exceptions.
- * For example, the method [java.io.File.getCanonicalFile] throws an [java.io.IOException]
- * which can be handled as follows:
- * ```
- * stream.map(Unchecked.function(file -&gt; file.getCanonicalFile())
- * ```
- *
- * Each method accepts a functional interface that is defined to throw [Throwable].
- * Catching `Throwable` means that any method can be wrapped.
- * Any `InvocationTargetException` is extracted and processed recursively.
- * Any [Error] or [RuntimeException] is re-thrown without alteration.
- * Any other exception is wrapped in a [RuntimeException].
- */
 internal object Unchecked {
-
-    /**
-     * Wraps a block of code, converting checked exceptions to unchecked.
-     * ```
-     * Unchecked.wrap(() -&gt; {
-     * // any code that throws a checked exception
-     * }
-     * ```
-     *
-     * If a checked exception is thrown it is converted to a [RuntimeException].
-     *
-     * @param <T>   the type of the result
-     * @param block the code block to wrap
-     * @return the result of invoking the block
-     * @throws RuntimeException if an exception occurs
-     </T> */
-    @JvmStatic
-    fun <T> wrap(block: CheckedSupplier<T>): T {
```

Comment:
We don't need these anymore, and arguably, we probably shouldn't have them.

Reply:
These are a hangover from the Java days & we're nearly at the point where they can be fully removed..